app/models/favorites/favorites.ts

```diff
@@ -24,27 +27,50 @@ export const FavoritesModel = types
       if (Platform.OS === "ios") {
         getIsWatchAppInstalled().then((isInstalled) => {
           if (isInstalled) {
-            this.updateAppleWatchFavorites()
             isWatchAppInstalled = true
           }
+
+          this.updateFavorites()
         })
+      } else {
+        this.updateFavorites()
       }
     },
-    updateAppleWatchFavorites() {
-      const appContext: WatchPayload = {}
-      self.routes.forEach((route, index) => {
-        appContext[index] = `originId:${route.originId},destinationId:${route.destinationId}`
-      })
-      updateApplicationContext(appContext)
+    updateFavorites() {
+      if (isWatchAppInstalled) {
+        const appContext: WatchPayload = {}
+        self.routes.forEach((route, index) => {
+          appContext[index] = `originId:${route.originId},destinationId:${route.destinationId}`
+        })
+        updateApplicationContext(appContext)
+      }
```

Comment:
I think it's still nice to have it as a separated method under it's own name, for clarity

Then maybe create a new method for the shortcuts with a relevant name (e.g. "syncShortcuts")